### PR TITLE
DOCS-1808: Update credit card errors

### PR DESCRIPTION
**Pull request by FrancescaNCook**
Do the changes meet MultiSafepay Docs' Definition of Done?
- Style guide, tone of voice, and naming conventions
- Tested
- Reviewed on mobile, tablet and desktop screens
- Notified stakeholders
- Pass GitHub Actions

### CODE REVIEW

File: content/payments/about-payments/card-errors.md [hunk 1]
```diff
@@ -7,149 +7,266 @@ short_description: "Reason codes for credit card errors."
...
+The transaction was declined because the cardholder's account is closed or blocked.  
 
-- Ask the cardholder to contact the issuer. Once resolved, try again.
-- If not resolved, ask the cardholder for another card, or to use a different payment method.
+Cardholder: Contact the issuer.  
+Merchant: Do **not** reattempt. The issuer will never approve. 
 
-#### 05: Do not honour / Card declined by issuer 
-The issuer has flagged a problem with the card and advises you not to accept it.  
+#### 05: Do not honour
```
Comment (Sarah-MultiSafepay) [inline content/payments/about-payments/card-errors.md:37]:
```suggestion
#### 05: Do not honor 
```

File: content/payments/about-payments/card-errors.md [hunk 2]
```diff
@@ -7,149 +7,266 @@ short_description: "Reason codes for credit card errors."
...
 
-Email <support@multisafepay.com>
+#### 58: Transaction not permitted to acquirer/terminal
+Merchant: Your MultiSafepay account is not set up for this payment method or type of transaction. Email <support@multisafepay.com>
 
-#### 62: Restricted card
-**Either** your MultiSafepay account is not set up for this payment method or type of transaction.  
+#### 59: Suspected fraud (Visa)
+Cardholder: Contact the issuer.  
+Merchant: If the cardholder confirms, reattempt.
```
Comment (Sarah-MultiSafepay) [inline content/payments/about-payments/card-errors.md:137]: Not sure what you mean by "the cardholder confirms"? What are they expected to confirm?

File: content/payments/about-payments/card-errors.md [hunk 3]
```diff
@@ -7,149 +7,266 @@ short_description: "Reason codes for credit card errors."
...
+#### 92: Unable to route the transaction
+There was a technical destination error in the issuer's system.
 
 #### 93: Transaction cannot be completed
-There is a violation on the cardholder's account and the issuer declined the transaction. 
+There is a temporary or permanent restriction on the cardholder's account, e.g.: 
+
+- Gambling
+- Unauthorized card-not-present transaction
+- No 2 factor authentication
```
Comment (Sarah-MultiSafepay) [inline content/payments/about-payments/card-errors.md:220]:
```suggestion
- No two-factor authentication  
```

File: content/payments/about-payments/card-errors.md [hunk 4]
```diff
@@ -7,149 +7,266 @@ short_description: "Reason codes for credit card errors."
...
 
-Email <support@multisafepay.com>
+#### 58: Transaction not permitted to acquirer/terminal
+Merchant: Your MultiSafepay account is not set up for this payment method or type of transaction. Email <support@multisafepay.com>
 
-#### 62: Restricted card
-**Either** your MultiSafepay account is not set up for this payment method or type of transaction.  
+#### 59: Suspected fraud (Visa)
+Cardholder: Contact the issuer.  
+Merchant: If the cardholder confirms, reattempt.
```
Comment (FrancescaNCook) [inline content/payments/about-payments/card-errors.md:137]:
```suggestion
Merchant: If the cardholder confirms that the issue is resolved, reattempt.
```

**Pull request merged**